Done changes in data saving and ui

Ritika8081 opened the pull request:
- [x] buttons alignment 
- [x] data saving in indexed db 
- [x] grid/list view
- [x] stop data on screen
- [x] file download and delete before start recording again

Amanmahe committed:
warning color changed and files popover button changed

Amanmahe committed:
updated code

Ritika8081 committed:
removed gap between grids,pause button and coverting 17 bytes into int

Ritika8081 committed:
added functionalty of pause dataon screen and added indexed db for browser storage

Ritika8081 committed:
download and delete zip file from indexeddb

Ritika8081 committed:
updated active/inactive feature of buttons

Amanmahe committed:
Data Handling for new Arduino code

Ritika8081 committed:
updated button alignment and updated indexeddb data formate

Ritika8081 committed:
updated autoscale button

Amanmahe committed:
changed in indexing of data and counter

Ritika8081 committed:
added some necessary comments

Ritika8081 committed:
Fix merge conflicts

Ritika8081 commented:
I have included aman's changes in this pull request #10 .

Ritika8081 committed:
changed software name, bydefault grid view and removed file popover

akadeepesh commented on .env:
```diff
@@ -0,0 +1 @@
+NEXT_PUBLIC_VERSION=2.0.0
```
Please create .env.local file instead of .env, and make sure to not push it on github.

akadeepesh commented on package.json:
```diff
@@ -29,6 +29,7 @@
     "@types/w3c-web-serial": "^1.0.6",
     "class-variance-authority": "^0.7.0",
     "clsx": "^2.1.1",
+    "dotenv": "^16.4.5",
```
Remove this package, we don't need this.

akadeepesh commented on src/components/Contributors.tsx:
```diff
@@ -12,7 +12,12 @@ import { Dialog, DialogContent, DialogTrigger } from "./ui/dialog";
 import { CircleAlert } from "lucide-react";
 import { Button } from "./ui/button";
 import Link from "next/link";
+import dotenv from "dotenv";
```
Remove this, you can access NEXT_EXAMPLE variable from env like process.env.NEXT_EXAMPLE. variable name can be anything this is just an example.

akadeepesh commented on src/components/Contributors.tsx:
```diff
@@ -62,6 +67,7 @@ const Contributors = () => {
         <DialogContent onOpenAutoFocus={(e) => e.preventDefault()}>
           <Card className="border-none -m-5">
             <CardHeader>
+              <p>Chords:{Version}</p>
```
I recommend to put software name in env as well

Ritika8081 replied:
I have done changes, please review.

akadeepesh commented:
Mark resolve conversation after doing changes.

Ritika8081 committed:
ui releted changes

- updated ui of logo and redirection
-updated auto scale button
-updated  version
- added comments

Ritika8081 committed:
resolved issues

akadeepesh commented:
I'm merging this to main :+1:
